scripts/cluster/create_multinode_cluster.go

```diff
@@ -71,43 +73,22 @@ func CreateMultinodeCluster(stockContainerd string) error {
 	return nil
 }
 
-// Create kubelet service on master node
-func CreateMasterKubeletService() error {
```

Comment:
Don't we need need it? Does it automatically attach to `unix:///run/containerd/containerd.sock`? That's the most important part here. But also, we are loosing the timeout and verbosity parameters. Do we need them?

Reply:
Because `kubeadm` starts `kubelet` process automatically, hence there is no need to start `kubelet` manually. Regarding the `containerd` socket, this is now provided as argument to `kubeadm init` command.

![image](https://github.com/vhive-serverless/vHive/assets/16463659/b511184f-71a2-4a7c-9859-cfc3939c06e1)

@leokondrashov: What is the socket path for Firecracker? Should we make this be configurable?

Reply:
This function doesn't create the kubelet, but rather sets it up with parameters. The socket on the master node doesn't need to be configurable since we use Firecracker only for user containers. But the verbosity and runtime-request-timeout set inside might be used for something.

Reply:
Hmm. You are right. 

For the `runtime-request-timeout` parameter, it seems to me we can safely remove this (see [DOCS](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)). This should mean that each request from the control plane to the kubelet should be executed within 2 minutes (default), rather than 15 minutes.

The `container-runtime` has been deprecated since v1.24 according to [this](https://github.com/cri-o/cri-o/blob/main/tutorials/kubernetes.md).